pkg/util/mocks/mock_arm/arm.go

```diff
@@ -9,7 +9,6 @@ import (
 	reflect "reflect"
 
 	gomock "github.com/golang/mock/gomock"
-
```

Comment:
This generated diff is invalid - you can either run `goimports -local=github.com/openshift/openshift-azure -e -w pkg` to fix it or wait for https://github.com/openshift/openshift-azure/pull/640 to merge, rebase on top, and rerun `go generate ./...`